source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -268,7 +271,8 @@ enum class Spacing {
 
 enum class ActionsOrientation {
     Vertical = 0,
-    Horizontal
+    Horizontal,
+    None,
```

Comment:
feel weird to have None as orientation type

Reply:
I use this when an orientation hasn't been explicitly set on an ActionSet, so that i know to fall back on whatever's in the host config. There's some precedent for this (image size also has a "none")

---
In reply to: [178390247](https://github.com/Microsoft/AdaptiveCards/pull/1312#discussion_r178390247) [](ancestors = 178390247)

Reply:
I could change it to Default though, that might seem less weird.

---
In reply to: [178391172](https://github.com/Microsoft/AdaptiveCards/pull/1312#discussion_r178391172) [](ancestors = 178391172,178390247)

Reply:
no, it makes sense to grab one from host config when it's set None.

---
In reply to: [178391349](https://github.com/Microsoft/AdaptiveCards/pull/1312#discussion_r178391349) [](ancestors = 178391349,178391172,178390247)